README.md

```diff
@@ -16,91 +16,43 @@ Once Docker is installed, run this to bootstrap the containers:
-
-### `npm start`
-
-Runs the app in the development mode.<br>
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-
-The page will reload if you make edits.<br>
-You will also see any lint errors in the console.
-
-### `npm test`
-
-Launches the test runner in the interactive watch mode.<br>
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `npm run build`
-
-Builds the app for production to the `build` folder.<br>
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.<br>
-Your app is ready to be deployed!
+Finally, you will need to set up an environment file containing various information and credentials. The file should be called `.env` and be located at the root of the project. The format is the familiar one for such files, with KEY=VALUE entries, one per line:
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `npm run eject`
-
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
-
-### Code Splitting
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/code-splitting
+```sh
+REACT_APP_MASTER_CURRICULUM_FOLDER_ID=1wEfbo0L404VKNFpiOtOyQvllfiTs7CnO
+REACT_APP_BUGSNAG_API_KEY=14385c1ebd16bc84362e6ce4e3f14e92
+```
 
-### Analyzing the Bundle Size
+Aside from the entries above, which you can use as-is, you will need the following:
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size
+* `REACT_APP_FIREBASE_API_KEY`: You can get this from the [Firebase console](https://console.firebase.google.com/), or by running `d/yarn.functions firebase setup:web`
+* `REACT_APP_FIREBASE_PROJECT_ID` is the ID of the Firebase project you created earlier.
```

Comment:
Perhaps another mention of where exactly to find this

Reply:
OK